README.md

```diff
@@ -2,31 +2,79 @@
 
-**Note that image naming has changed recently**. We now use different names for images with different purposes, while previously we were using tagging to distinguish between them. CIs and scripts will need to be updated accordingly on updates after v0.23.1.
+**Image naming has changed recently.**
 
-Biggest naming change: the app generator gem used to be called `decidim` and is now called `decidim-generator`. 
+We now use different names for images with different purposes, while previously we were using tagging to distinguish between them. CIs and scripts will need to be updated accordingly on updates after v0.23.1.
+
+**Biggest naming change:** 
+
+The app generator gem used to be called `decidim` and is now called `decidim-generator`. There's now a new `decidim` image for a fully functioning generated Decidim app.
 
 Here's the complete list of images and their purposes:
 
 ## Images available
 
-- `decidim-generator:latest` or `decidim-generator:<version>` (eg: `decidim-generator:0.23.1`): the [decidim gem](https://rubygems.org/gems/decidim) with all necessary environment for running it.
-- `decidim-test:latest` or `decidim-test:<version>` (eg: `decidim-test:0.23.1`): the above gem environment plus tooling for testing.
-- `decidim-dev:latest` or `decidim-dev:<version>` (eg: `decidim-dev:0.23.1`): the above plus more configuration for running local dev environment.
-- `decidim:latest` or `decidim:<version>` (eg: `decidim:0.23.1`): actual generated Decidim app to be run locally.
+### decidim
+
+Decidim app pre-generated with core modules.
+
+Tagged as `decidim:latest` or `decidim:<version>` (eg: `decidim:0.23.1`). 
+
+### decidim-generator
+
+The [decidim gem](https://rubygems.org/gems/decidim) with all necessary environment for running it.
+
+Tagged as `decidim-generator:latest` or `decidim-generator:<version>` (eg: `decidim-generator:0.23.1`).
+
+### decidim-test
+
+The above gem environment plus tooling for testing.
+
+Tagged `decidim-test:latest` or `decidim-test:<version>` (eg: `decidim-test:0.23.1`).
+
+### decidim-dev
+
+The above plus more configuration for running local dev environment.
+
+Tagged `decidim-dev:latest` or `decidim-dev:<version>` (eg: `decidim-dev:0.23.1`).
 
-## Using the decidim (app) image
+## Hello World with docker-compose
 
-With this image you can run a pre-generated Decidim app:
+This repo includes a [docker-compose.yml](docker-compose.yml) file with:
 
+- a Decidim service (using the `decidim:latest` image)
+- a Postgres service
+- a Redis service
 
-### Locally
+By cloning the repo and then running `docker-compose up`, you'll get a fully functional Decidim app complete with seed data, accessible at http://localhost:3000.
+
+```bash
+git clone git@github.com:decidim/docker.git decidim-docker
+cd decidim-docker
+docker-compose up
+```
+It'll take a couple of minutes to run through all migrations (rake db:schema:load isn't working, currently) and seeds. At the end you should see:
```

Comment:
`rake db:schema:load` won't probably work because we don't have the `db/schema.rb` file when we create the app for the first time. That file is generated after each migration, so until we run them we won't have it 

(unless we copy if from somewhere else, but I don't think that's encouraged)

Reply:
Yeah, that’s what I meant by “not currently working”, but yours is a much clearer explanation :) I’ll include it in the comment.

I’ve been thinking about the schema.rb issue, wondering if it’d possible to pre-generate it during the docker build during a throwaway build phase. I might play with that and see where it leads, if anywhere.

Reply:
Done.